WalletWasabi.Gui/Tabs/EncryptionManager/EncryptMessageView.xaml

```diff
@@ -0,0 +1,21 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui" Design.Width="600">
+  <controls:GroupBox Title="{Binding Title}" BorderThickness="0" Classes="docTabContainer">
+    <Grid Classes="content" Margin="0 40 0 0">
+      <Grid.RowDefinitions>
+        <RowDefinition Height="*" />
+        <RowDefinition Height="Auto" />
+        <RowDefinition Height="*" />
+        <RowDefinition Height="Auto" />
+      </Grid.RowDefinitions>
+      <controls:ExtendedTextBox Grid.Row="0" Text="{Binding PlainMessage}" Watermark="Message" UseFloatingWatermark="True" TextWrapping="Wrap" AcceptsReturn="True" />
+      <controls:ExtendedTextBox Grid.Row="1" Text="{Binding PublicKey}" Watermark="Public key of recipient" UseFloatingWatermark="True" />
+      <!--<controls:NoparaPasswordBox Grid.Row="2" Password="{Binding Password}" Watermark="Password" UseFloatingWatermark="True" />-->
+      <controls:ExtendedTextBox Grid.Row="2" Text="{Binding EncryptedMessage}" Watermark="EncryptedMessage" TextWrapping="Wrap" IsReadOnly="True" AcceptsReturn="True" UseFloatingWatermark="True" />
```

Comment:
There is a missing stape in the Watermark, it says "EncryptedMessage" instead of "Encrypted Message".